### ISSUE TITLE
Descriptive page title: another example for consideration (c4a8a4)

### ISSUE THREAD

**Issue by spanchang** [act-rules/act-rules.github.io#1006]
consider adding another example for a situation like the one below:
Product page has a link for enlarged view of the product.
When activated, the link opens up a new page (HTML) with the enlarged view of the product and a close button at the bottom.
Yes the image needs an alt hopefully that identifies the product. Or it could be set to 'Enlarged view' which may not be very helpful.
But what if the page title  simply says 'Enlarged view'  or 'Expanded view'? 
That will not satisfy SC 2.4.2. Can this be detected in an automated fashion?
It is not just products: sometimes it is   for pictures of people like team members etc.
I have been flagging it manually when I have encountered one over the years. 
Thanks,

**Comment by jkodu** [comment]
Rule in question - https://act-rules.github.io/rules/c4a8a4

**Comment by spanchang** [comment]
Can this issue be made auto-detectable by atool?

**Comment by patrickhlauke** [comment]
needs human judgement to say what is and isn't "descriptive" enough

**Comment by EmmaJP** [comment]
Yep probably not auto-detectable, but an example with an image with good alt text would be a sensible additional Failed Example.

**Comment by Jym77** [comment]
```html
<html>
<head><title>Bad title</title></head>
<body><img src="..." alt="Good description" /></body>
</html>
```

**Comment by spanchang** [comment]
A failed example would be one with a page title that is "Enlarged View" and a good example would be "Enlarged View - <product name and identifier> / <Person name and designation>". This will fail SC 2.4.2 regardless of whether the text alternative for the image is appropriate.
Thanks

**Comment by WilcoFiers** [comment]
I'm not sure this needs to be done. If someone wants to pick this up still, open a pull request. I don't think this needs to stay open.
